docs/articles/scaleout.md

```diff
@@ -0,0 +1,168 @@
+# Scale-out clustering
+
+> :point_right: A cluster of zot instances can be easily scaled with no repo-specific intelligence in the load balancing scheme, using:
+>
+> -   Stateless zot instances to simplify scale out
+> -   Shared remote storage
+> -   zot release v2.1.0 or later
+
+Beginning with zot release v2.1.0, a new "scale-out" architecture greatly reduces the configuration required when deploying large numbers of zot instances. As before, multiple identical zot instances run simultaneously using the same shared reliable storage, but with improved scale and performance in large deployments. A highly scalable cluster can be architected by automatically sharding based on repository name so that each zot instance is responsible for a subset of repositories.
+
+In a cloud deployment, the shared backend storage (such as AWS S3) and metadata storage (such as DynamoDB) can also be easily scaled along with the zot instances.
+
+> :pencil2: For high availability clustering with earlier zot releases, see [zot Clustering](clustering.md).
+
+## Prerequisites
+
+For easy scaling of instances (replicas), the following conditions must be met:
+
+- All zot replicas must be running zot release v2.1.0 (or later) with identical configurations.
+- All zot replicas in the cluster use remote storage at a single shared S3 backend. There is no local caching in the zot replicas.
+- Each zot replica in the cluster has its own IP address, but all replicas use the same port number.
+- The URI format sent to the cluster must be /v2/<repo\>/<manifest\>:<tag\>
```

Comment:
Not sure if this is a pre-requisite as such.
Only the requests having /v2/<repo> would be proxied, but it's not a pre-requisite to scale instances as such. The other APIs continue to work as usual as the storage is shared.

Reply:
Removed line 22 ("- The URI format sent to the cluster must be /v2/<repo\>/<manifest\>:<tag\>")